### PR TITLE
OSDOCS-10540: Add IPsec telementry field

### PR DESCRIPTION
**Pull request by jab-rh**
Possible backport to 4.15 eventually.

Version(s): 4.16+

Issue: https://issues.redhat.com/browse/OSDOCS-10540

Link to docs preview: https://76698--ocpdocs-pr.netlify.app/openshift-enterprise/latest/support/remote_health_monitoring/showing-data-collected-by-remote-health-monitoring.html

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by jab-rh** [comment]
@JoshSalomon, how does this look? Thanks!

**Review by xenolinux — COMMENTED** [review]
/remove-label peer-review-needed
/remove-label peer-review-in-progress
/label peer-review-done

LGTM

File: modules/telemetry-showing-data-collected-from-the-cluster.adoc [hunk 1]
```diff
@@ -183,5 +183,6 @@ openshift-monitoring -o jsonpath="{.spec.host}")/federate \
 --data-urlencode 'match[]={__name__="namespace:noobaa_usage:max"}' \
 --data-urlencode 'match[]={__name__="namespace:noobaa_system_health_status:max"}' \
 --data-urlencode 'match[]={__name__="ocs_advanced_feature_usage"}' \
---data-urlencode 'match[]={__name__="os_image_url_override:sum"}'
+--data-urlencode 'match[]={_name_="os_image_url_override:sum"}' \
```
Comment (huiran0826) [inline modules/telemetry-showing-data-collected-from-the-cluster.adoc:186]: Not sure what is this about, it is not ipsec related.

File: modules/telemetry-showing-data-collected-from-the-cluster.adoc [hunk 2]
```diff
@@ -183,5 +183,6 @@ openshift-monitoring -o jsonpath="{.spec.host}")/federate \
...
+--data-urlencode 'match[]={_name_="openshift:openshift_network_operator_ipsec_state:info"}'
```
Comment (huiran0826) [inline modules/telemetry-showing-data-collected-from-the-cluster.adoc:187]: {__name__="openshift:openshift_network_operator_ipsec_state:info"}, it seems `__name__`, not `_name_`
@JoshSalomon  can you help confirm?

Comment (jab-rh) [reply modules/telemetry-showing-data-collected-from-the-cluster.adoc:187]: @huiran0826, so this works:

```
curl -G -k -H "Authorization: Bearer $(oc whoami -t)" \
                  https://$(oc get route prometheus-k8s-federate -n \
              openshift-monitoring -o jsonpath="{.spec.host}")/federate \
                  --data-urlencode 'match[]={__name__="openshift:openshift_network_operator_ipsec_state:info"}'
# TYPE openshift:openshift_network_operator_ipsec_state:info untyped
openshift:openshift_network_operator_ipsec_state:info{is_legacy_api="false",mode="Disabled",instance="",prometheus="openshift-monitoring/k8s",prometheus_replica="prometheus-k8s-1"} 1 1719242955034
```

**Pull request merged**